website/docs/how-to/multiple-portals-multiple-web-apps.md

```diff
@@ -10,25 +10,54 @@ In some cases, it makes sense to have different teams handle different portals i
 
 ## Declaring Multiple Portals
 
-Setting up multiple Portals is as easy as declaring another Portal in the PortalManager. No further code is neccessary and each Portal will function independently.
+### iOS
+
+Setting up multiple Portals is as easy as initializing another Portal. 
```

Comment:
I know you didn't introduce this but I just noticed we use "is as easy as" and "simply" in a few places around here. We could address it in another PR.

Reply:
Yeah, I agree, there's a lot of "Slap Chop" vibes going on with the wording in a lot of places

Reply:
in case anyone forgot about Slap Chop https://www.youtube.com/watch?v=rUbWjIKxrrs